src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -105,7 +105,7 @@ const mainSelector = createSelector(
       // Add the My VA link to default links.
       {
         className: 'my-va-top-nav',
-        href: '/my-va/',
+        href: 'https://va.gov/my-va/',
```

Comment:
nit - the `www` would prevent a middleman redirect

Reply:
don't worry about it though since CI takes so long

Reply:
added